packages/ui/primitives/document-flow/add-fields.tsx

```diff
@@ -428,6 +430,31 @@ export const AddFieldsFormPartial = ({
           return;
         }
 
+        if (duplicateAllPages) {
+          const pages = Array.from(document.querySelectorAll(PDF_VIEWER_PAGE_SELECTOR));
+          const totalPages = pages.length;
+
+          for (let pageNumber = 1; pageNumber <= totalPages; pageNumber++) {
+            if (pageNumber !== lastActiveField.pageNumber) {
+              const newField: TAddFieldsFormSchema['fields'][0] = {
```

Comment:
this is more like nitpicking, but maybe we could extract this into a constant and re-use it. but feel free to ignore this comment.

Reply:
>  we could extract this into a constant and re-use it

could you clarify with an example?